axelrod/ecosystem.py

```diff
@@ -0,0 +1,38 @@
+class Ecosystem(object):
+    """Create an ecosystem based on the payoff matrix from an Axelrod tournament."""
+
+    def __init__(self, pmatrix):
+
+        self.pmatrix = pmatrix
```

Comment:
Yeah: this is an awesome way of doing this. This whole way of doing it (including the rest of it) is very elegant. Not sure my applause carries much weight but :clap: